src/drivers/github.js

```diff
@@ -396,45 +398,74 @@ class Github {
    * @param {{ pullRequestId: number, base: string }} param0
    * @returns {Promise<void>}
    */
-  async prAutoMerge({ pullRequestId, base }) {
+  async prAutoMerge({
+    pullRequestId,
+    mergeMode,
+    mergeMessage = undefined,
+    base
+  }) {
     const octo = octokit(this.token, this.repo);
     const graphql = withCustomRequest(octo.request);
-
+    const { owner, repo } = this.ownerRepo();
+    const [commitHeadline, commitBody] =
+      mergeMessage === undefined ? [] : mergeMessage.split(/\n\n(.*)/s);
```

Comment:
```suggestion
      (mergeMessage || '').split(/\n\n(.*)/s);
```

Reply:
@DavidGOrtega, what you propose isn't functionally equivalent when `mergeMessage` is undefined:

#### Before

```js
commitHeadline === undefined
commitBody == undefined
```

#### After

```js
commitHeadline === ''
commitBody == undefined
```

Reply:
If you're allergic to `undefined` comparisons, we can do this:

```suggestion
      mergeMessage ? mergeMessage.split(/\n\n(.*)/s) : [];
```

Reply:
@0x2b3bfa0 sorry I do not follow you. Could you please elaborate?

if speaking about 
```js
(mergeMessage || '').split(/\n\n(.*)/s);
```
vs
```js
mergeMessage === undefined ? [] : mergeMessage.split(/\n\n(.*)/s);
```

it should be equivalent (shorter). Indeed if we were setting mergeMessage with '' we would just only need

```js
mergeMessage.split(/\n\n(.*)/s);
```

Reply:
Ah! its true the empty string. However probably for the outcome it would be the same.
In any case and to be exact to your desire 

```suggestion
      mergeMessage ? mergeMessage.split(/\n\n(.*)/s) : [];
```

If not if you setup mergeMessage with empty string your will endup with the same outcome, an empty string in commitHeadline